pkg/eyecandy/emoji.go

```diff
@@ -0,0 +1,26 @@
+package eyecandy
+
+import (
+	"fmt"
+	"github.com/kyokomi/emoji/v2"
+	"regexp"
+)
+
+func ESPrintf(emojisDisabled bool, format string, v ...interface{}) string {
+	if emojisDisabled {
```

Comment:
This kind of sucks, having to pass this flag all the time. Isnt there a way we can directly obtain it here from the settings or environment?

Reply:
Agree here.

Reply:
I know, I found 2 ways, either moving the settings into global variable, which sucks in other ways or adding the ESPrint functions to the environment.go, where they clearly do not belong. So I opted for the least sucking way :)

Reply:
So how do we resolve this? Keep it this way for now & merge?